UnityProject/Assets/Scripts/Items/Food/HealableJobEdible.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+
+public class HealableJobEdible : Edible
+{
+	[SerializeField, Tooltip("The amount of brute damage healed for a player who has a specfied role.")]
+	private int healBruteDamage = 5;
+	public List<JobType> healableJobs;
```

Comment:
this public isn't used anywhere but in this class. Make it private with serialized field attribute just like the int for healing amount.